### ISSUE TITLE
Decision to use nawk

### ISSUE THREAD

**Issue by maflister** [i2bc/SURFMAP#4]
Hello,

I wanted to know if you chose nawk for a specific feature, or if gawk, mawk, etc. will produce the same result. I.e., is there a nawk specific feature or is it just the version of awk you had on your system during development?

Thanks!

**Comment by nchenche** [comment]
Hello Matt,

The use of nawk does not rely on us, developers of SURFMAP, but rather on the M. Sanner team who developed MSMS, a program used by (and included in) SURFMAP. However, to answer your question, I don't think there is any specific feature that would make the use of nawk mandatory. 

Nawk is used in MSMS by a script called `pdb_to_xyzr`. I've just made a test after having edited this file in which I replaced _nawk_ with _awk_. After this change I run the following command `./pdb_to_xyzr 1crn.pdb` which gave me the same output I obtained with nawk. 

To recap, you can try to use awk if you want. You'll need to edit the file `pdb_to_xyzr` located in the MSMS directory (present at the root repository of SURFMAP) and then replace nawk with awk.

Hope this will help.

PS: I haven't tried with another awk version (differences between some awk versions are accessible [here](https://www.thegeekstuff.com/2011/06/awk-nawk-gawk/)
